util_scripts/Change_Channel_Names.py

```diff
@@ -119,22 +119,29 @@ def getLcIdsList(self, query):
         lc_ids = [lc_id[0] for lc_id in lc_ids]
         return [lc_id.getValue() for lc_id in lc_ids]
 
-    def updateImageNames(self, image_list):
-        image_list = self.update_service.saveAndReturnArray(image_list)
-
-    def removeLCsFromList(self, image, lc_ids_list):
-        image_noc = image.getPrimaryPixels().getSizeC().getValue()
-        for c in range(image_noc):
-            lc_id = image.getPrimaryPixels().getChannel(c). \
-                getLogicalChannel().getId().getValue()
-            lc_ids_list.remove(lc_id)
+    def removeLCsFromList(self, lc_list, lc_ids_list):
+        for lc in lc_list:
```

Comment:
Just for clarity you might want to work with one or more `set` instances here so you can just ask for an intersection via `a & b`.